### PR TITLE
App Submission: WBO

### PR DESCRIPTION
**Pull request by IMPranshu**
# App Submission

### WBO
...

### 256x256 SVG icon
https://github.com/IMPranshu/svg-logo/blob/main/wbo.svg

...

### Gallery images
Screenshot from 2022-11-11 00-58-21

wbo4
wbo3
wbo2
wbo1

...

### I have tested my app on:
- [ ] [Umbrel dev environment](https://github.com/getumbrel/umbrel-dev)
- [x] [Umbrel OS on a Raspberry Pi 4](https://github.com/getumbrel/umbrel-os)
- [x] [Custom Umbrel install on Linux](https://github.com/getumbrel/umbrel#-installation)

### CODE REVIEW

**Comment by lovasoa** [comment]
Hi ! I'm Ophir, the main developer of WBO, and I'd be glad to see it featured on the Umbrel app store :slightly_smiling_face:

File: wbo/docker-compose.yml [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+  app_proxy:
+    environment:
+      APP_HOST: wbo
+      APP_PORT: 80
+
+  server:
+    container_name: wbo
+    volumes:
+      - ${APP_DATA_DIR}/wbo-boards:/opt/app/server-data
+    image: impranshu/wbo:1@sha256:1458870c58a319638bcf7d474b76fe4b2eafb1020a97444680345a60d0e7896c
```
Comment (lovasoa) [inline wbo/docker-compose.yml:13]: Shouldn't we put the official docker image here ?

https://hub.docker.com/r/lovasoa/wbo

Comment (IMPranshu) [reply wbo/docker-compose.yml:13]: Yeah. Absolutely. At that time I didn't find the official docker image so I made a docker image and went ahead. Will be modifying the code and opening the PR soon. Thanks

**Comment by IMPranshu** [comment]
@nevets963 Can you please open the PR?? I have permission from the dev for publishing.

File: wbo/docker-compose.yml (hunk 1 above)
Comment (IMPranshu) [reply wbo/docker-compose.yml:13]: I have updated the link and verified that it's working fine. Thanks

Comment (lovasoa) [reply wbo/docker-compose.yml:13]: I still see `impranshu/wbo` instead of `lovasoa/wbo`

**Comment by lovasoa** [comment]
Please wait for the docker image to point to `lovasoa/wbo` before publishing.

**Comment by IMPranshu** [comment]
I have already updated the docker image. I think this PR is closed that's why new commits are not showing up. I have opened a new PR https://github.com/getumbrel/umbrel-apps/pull/663, please verify the changes @lovasoa. Thanks

**Pull request closed without merging**